test/com/engeto/CountryExtractorTest.java

```diff
@@ -0,0 +1,41 @@
+package com.engeto;
+
+import kong.unirest.GetRequest;
+import kong.unirest.HttpResponse;
+import kong.unirest.JsonNode;
+import kong.unirest.Unirest;
+import kong.unirest.json.JSONArray;
+import kong.unirest.json.JSONObject;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+public class CountryExtractorTest {
+    @Test
+    public void extractCountriesTest() {
+        GetRequest request = Unirest.get("http://jsonvat.com/");
+        HttpResponse<JsonNode> jsonResponse = request.asJson();
+        JsonNode json = jsonResponse.getBody();
+        JSONObject jsonObject = json.getObject();
+
+        JSONArray list = jsonObject.getJSONArray("rates");
+        ArrayList<Country> countries = new ArrayList<>();
+
+        for (Object elements : list) {
```

Comment:
Opytaj sa, ked niekto zmeni (a rozbije) CountryExtractor, prejavi sa to nejak v tomto testu?
Davaj pozor na tento koment, tu je zakopane najvacsia chyba v projektu.

Reply:
Jo, tohle jsem zkopal, dekuji, ted uz by to snad melo byt ok.